contracts/solana/programs/agreement/src/lib.rs

```diff
@@ -1,18 +1,17 @@
 //! Agreement Program Test Suite
```

Comment:
This file is starting to get huge. Maybe it is time to separate logic into other files and import them on the main lib.rs